### PR TITLE
Feature/dfam

### PR DESCRIPTION
**Pull request by jerowe**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Creating a recipe for DFAM.

http://dfam.org/help/tools

### CODE REVIEW

**Comment by bgruening** [comment]
Here is way more than one recipe, not that it hurts, but I guess this was not intended?

**Comment by jerowe** [comment]
Nope. Except for dfam all those recipes were already merged.

**Comment by jerowe** [comment]
There we go. All better now.

**Comment by bgruening** [comment]
:)

File: recipes/dfam/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+package:
+  name: dfam
+  version: '2.0'
+
+requirements:
+  run:
+      - perl-threaded
+      - hmmer
+      - repeatmasker
+
```
Comment (bgruening) [inline recipes/dfam/meta.yaml]: You should include test for dfamscan.pl and the download-script. At least if these scripts are executable.

**Comment by jerowe** [comment]
I had conflicts I had to resolve.

https://github.com/bioconda/bioconda-recipes/pull/2017

**Pull request closed without merging**